packages/reaction-core/client/helpers/apps.js

```diff
@@ -165,8 +165,19 @@ function getReactionApps(optionHash) {
                 if (!registry.packageName) registry.packageName = app.name;
                 if (registry.enabled !== false) {
                   // TODO move this to a function and reuse with Template.dashboardHeader.helpers
-                  const registryLabel = registry.label ? registry.label.toCamelCase() : "";
-                  const i18nKey = `admin.${registry.provides}.${registryLabel}`;
+                  let registryLabel = "";
+                  let i18nKey;
+                  // first we check the default place for a label
+                  if (registry.label) {
+                    registryLabel = registry.label.toCamelCase();
+                    i18nKey = `admin.${registry.provides}.${registryLabel}`;
+                    // and if we don't find it, we are trying to look at first
```

Comment:
Hi, sure:) I did it on screenshots, and you could repeat it manually.
1. Put breakpoint [here](https://github.com/reactioncommerce/reaction/blob/development/packages/reaction-dashboard/client/templates/dashboard/shop/settings/settings.js#L51)
2. go inside `ReactionCore.Apps`
3. then run [here](https://github.com/reactioncommerce/reaction/blob/development/packages/reaction-core/client/helpers/apps.js#L168)
4. check `registry` object